samples/notebooks/python/01-basic-loading-the-kernel.ipynb

```diff
@@ -0,0 +1,188 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Basic Loading of the Kernel"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To run the notebooks we recommend using Poetry and starting a shell with a virtual environment\n",
+    "prepared to use SK. \n",
+    "\n",
+    "See [DEV_SETUP.md](../../../python/DEV_SETUP.md) for more information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m pip install -r requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import semantic_kernel as sk\n",
+    "import semantic_kernel.ai.open_ai as sk_oai"
```

Comment:
I would keep this consistent with the other notebooks and use `from semantic_kernel.ai.open_ai import AzureTextCompletion, OpenAITextCompletion`